multimedia/gst1-libav/Makefile

```diff
@@ -146,12 +146,27 @@ LIBAV_CONFIGURE_PROTOCOLS:=$(call FILTER_CONFIG,PROTOCOL,protocol,$(LIBAV_PROTOC
 REAL_CPU_TYPE:=$(firstword $(subst +, ,$(CONFIG_CPU_TYPE)))
 
 CONFIGURE_ARGS += \
+	--disable-Bsymbolic \
+	--disable-fatal-warnings \
+	--disable-glib-asserts \
+	--disable-gobject-cast-checks \
+	--disable-gtk-doc-html \
+	--disable-valgrind \
 	--without-system-libav \
+	--with-pic \
```

Comment:
Use the toolchain logic for this instead?

Reply:
I've done tests before. fPIC vs. fpic on both ARM and MIPS make no difference in compiled size.

Also, some packages default to --with-pic. In which case having fPIC and fpic together makes no real sense.